packages/augur-tools/src/templates-template.ts

```diff
@@ -1,3 +1,148 @@
+export const REQUIRED = 'REQUIRED';
+// Market templates
+export const SPORTS = 'Sports';
+export const POLITICS = 'Politics';
+export const FINANCE = 'Finance';
+export const ENTERTAINMENT = 'Entertainment';
+export const CRYPTO = 'Crypto';
+export const USD = 'USD';
+export const USDT = 'USDT';
+export const EUR = 'EUR';
+
+// Market Subtemplates
+export const SOCCER = 'Soccer';
+export const AMERICAN_FOOTBALL = 'American Football';
+export const BASEBALL = 'Baseball';
+export const GOLF = 'Golf';
+export const BASKETBALL = 'Basketball';
+export const TENNIS = 'Tennis';
+export const HOCKEY = 'Hockey';
+export const HORSE_RACING = 'Horse Racing';
+export const US_POLITICS = 'US Politics';
+export const WORLD = 'World';
+export const STOCKS = 'Stocks';
+export const INDEXES = 'Indexes';
+export const BITCOIN = 'Bitcoin';
+export const ETHEREUM = 'Ethereum';
+export const LITECOIN = 'Litecoin';
+export const BTC = 'BTC';
+export const ETH = 'ETH';
+export const LTC = 'LTC';
+export const NBA = 'NBA';
+export const NCAA = 'NCAA';
+export const NFL = 'NFL';
```

Comment:
nitpick: Might be nicer if you used enums for these.

Reply:
I'm use to having them split out, also these values are used in UI. I like to have them separated out.